src/components/Colors/Colors.tsx

```diff
@@ -0,0 +1,60 @@
+import { Color } from "./Color";
+export function Colors() {
+  const colorBrand = "bg-[#1AE19D]";
+  const colorWhite = "bg-[#FFFFFF]";
+  const colorDarkGrey = "bg-[#DADADA]";
+  const colorLightGrey = "bg-[#F1F1F1]";
+  const colorDarkBlue = "bg-[#151A26]";
+  const colorBasicBlue = "bg-[#1A202E]";
+  const colorLightBlue = "bg-[#2E374C]";
+  const colorWarning = "bg-[#FFA41D]";
+  const colorError = "bg-[#FF4F4F]";
```

Comment:
I believe these colors are already defined in our codebase. You can check here: presets/massa-station-preset.js

You don't need to redefine them once again. Just import from there (maybe it need an export there) and then use it here. Like that we have one, and only, source of truth.

WDYT!?

Reply:
This is outdated !